docs/src/modules/components/AppTableOfContents.js

```diff
@@ -64,11 +65,18 @@ const NavItem = styled(Link, {
       color: (theme.vars || theme).palette.primary[400],
     },
   };
+  let paddingLeft = '10px';
```

Comment:
```suggestion
  let paddingLeft = '12px';
```

Just to have everything being a multiple of either 4 or 6!